bundler/bundler.go

```diff
@@ -172,14 +173,15 @@ func (b *Bundler) BundleFromFile(bundleFile, keyFile string, flavor BundleFlavor
 		}
 	}
 
-	return b.BundleFromPEM(certsPEM, keyPEM, flavor)
+	return b.BundleFromPEMorDER(certsRaw, keyPEM, password, flavor)
 }
 
 // BundleFromPEM builds a certificate bundle from the set of byte
 // slices containing the PEM-encoded certificate(s), private key.
-func (b *Bundler) BundleFromPEM(certsPEM, keyPEM []byte, flavor BundleFlavor) (*Bundle, error) {
+func (b *Bundler) BundleFromPEMorDER(certsRaw, keyPEM []byte, password string, flavor BundleFlavor) (*Bundle, error) {
```

Comment:
For consistency between BundleFromPEM and BundleFromFile, the password should be the last parameter.